keps/sig-node/3857-rro-hostpath/kep.yaml

```diff
@@ -0,0 +1,46 @@
+title: Recursively read-only hostPath volumes
+kep-number: 3857
+authors:
+  - "@AkihiroSuda"
+owning-sig: sig-node
+#participating-sigs:
+#  - sig-aaa
+#  - sig-bbb
+status: implementable
+creation-date: 2023-02-09
+reviewers:
+  - TBD
+approvers:
+  - TBD
+
+#see-also:
+#  - "/keps/sig-aaa/1234-we-heard-you-like-keps"
+#  - "/keps/sig-bbb/2345-everyone-gets-a-kep"
+#replaces:
+#  - "/keps/sig-ccc/3456-replaced-kep"
+#
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.28"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.28"
+#  beta: "v1.XX"
+#  stable: "v1.XX"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: RecursivelyReadOnlyHostPath
```

Comment:
Is this really just about HostPath?  I think it's all volumets, right?

Reply:
Changed to `RecursivelyReadOnlyMounts`.